install-dev/controllers/http/process.php

```diff
@@ -100,11 +100,11 @@ public function process(): void
                     ->setProcessFOThemes(['classic'])
                     ->process();
                 $this->processConfigureShop();
-            } elseif (Tools::getValue('installModules') && (!empty($this->session->process_validated['configureShop']) || !$validateFixturesInstallation)) {
-                $this->processInstallModules();
-            } elseif (Tools::getValue('installTheme') && !empty($this->session->process_validated['installModules'])) {
+            } elseif (Tools::getValue('installTheme') && !empty($this->session->process_validated['configureShop'])) {
```

Comment:
Not sure it's necessary to invert those two, since they are independent use cases based on the request parameters their order shouldn't matter here The important is the order of the steps that you changed downer

Reply:
Yes it is because we check `$this->session->process_validated['configureShop']` before calling `installTheme`, then we check `installTheme` before calling `installModules`, etc.. 🙂

Reply:
Oh right! I didn't notice you updated the step that is supposed to be executed right before I get it now!